src/openms_gui/source/VISUAL/MISC/FilterableList.cpp

```diff
@@ -69,20 +69,38 @@ namespace OpenMS
 
     void FilterableList::setBlacklistItems(const QStringList& bl_items)
     {
+      /*
+       * Suppressing warning toSet() deprecated till Qt 5.14
+       */
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
       blacklist_ = bl_items.toSet();
+#pragma GCC diagnostic pop
       updateInternalList_();
     }
 
     void FilterableList::addBlackListItems(const QStringList& items)
     {
+      /*
+       * Suppressing warning toSet() deprecated till Qt 5.14
```

Comment:
this is not toSet()

Reply:
And I have done for only those lines against which compiler was warning

Reply:
sorry my bad :P